File: WebApi/Models/SettingsModel.cs
using System;
using System.Collections.Generic;

namespace WebApi.Models
{
    public class SettingsModel
    {
        public string Id {get;set;}
        public double? DeadlineTimeSpanInMiliseconds {get;set;}
        public double? PanicTimeSpanInMiliseconds {get;set;}
        public double? StartPanicForTimeSpanInMiliseconds {get;set;}
    }
}

File: AngularClient/src/app/modules/content/models/settings-model.ts
import { ISettingsModel } from '../@types/tasks';

export class SettingsModel{
    id: string;
    deadlineTimeSpanInMiliseconds: number;
    panicTimeSpanInMiliseconds: number;
    startPanicForTimeSpanInMiliseconds: number;

    constructor(data: ISettingsModel){
        this.id = data.id;
        this.deadlineTimeSpanInMiliseconds= data.deadlineTimeSpanInMiliseconds;
        this.panicTimeSpanInMiliseconds= data.panicTimeSpanInMiliseconds;
        this.startPanicForTimeSpanInMiliseconds= data.startPanicForTimeSpanInMiliseconds;
    }
}